src/librustc_mir/interpret/terminator.rs

```diff
@@ -19,7 +19,6 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         use rustc_middle::mir::TerminatorKind::*;
         match terminator.kind {
             Return => {
-                self.frame().return_place.map(|r| self.dump_place(*r));
```

Comment:
Hm, we might still want to dump the return value...

Reply:
Yeah, I added this back when the return value is copied when returning from the function.